src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -219,7 +219,8 @@ public void count(final String toggleName, boolean enabled) {
     }
 
     private Strategy getStrategy(String strategy) {
-        return strategyMap.containsKey(strategy) ? strategyMap.get(strategy) : UNKNOWN_STRATEGY;
+        Strategy foundStrategy = strategyMap.getOrDefault(strategy, config.getFallbackStrategy());
```

Comment:
Would it not be better to say that UNKNOWN_STRATEGY is the default fallback strategy, if not overridden?

Reply:
Yes. I'll change to that approach. I like that better since it means there will always be a fallback, but I held off changing too much.